### PR TITLE
Refactor and finish ansible playbooks, refactor and simplify terraform config

### PR DESCRIPTION
**Pull request by Greg-Szabo**

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+ansible-playbook -i inventory/digital_ocean.py install.yml -e tendermint_release_install=false -e basecoin_release_install=false
+```
+
+Alternatively you can change the variable settings in `group_vars/all`.
+
+## Other commands and roles
+
+There are few extra playbooks to make life easier managing your servers.
+
+* install.yml - the all-in-one playbook to install and configure tendermint + basecoin
```
Comment (melekes) [inline ansible/README.md]: what if the user just wants tendermint? (without basecoin)

**Comment by melekes** [comment]
❤️ 💚 💛  great work

2 questions:
1. Is there a section in readme explaining how to setup only tendermint. If there isn't, we should add it. Not all our users will want basecoin.
2. Do we want log files? Who's responsibility is to rotate log files? Maybe we should write to syslog by default?

**Comment by adrianbrink** [comment]
I second melekes on the first point. For example, it would be super cool to also enable an easy deployment of ethermint.

File: ansible/README.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+* python dopy package
+
+Head over to the [Terraform folder](https://github.com/tendermint/tools) for a description on how to get a DigitalOcean API Token.
+
+Optional for Amazon AWS instances:
+* Amazon AWS API access key ID and secret access key.
+
+The cloud inventory scripts come from the ansible team at their [GitHub](https://github.com/ansible/ansible) page. You can get the latest version from the contrib/inventory folder.
+
+## Ansible setup
```
Comment (adrianbrink) [inline ansible/README.md]: Should we also add explanation for setup on OSX, since it also supports ansible.

Comment (Greg-Szabo) [inline ansible/README.md]: What if I remove the ansible setup part and replace it with this link? [Ansible Installation](http://docs.ansible.com/ansible/intro_installation.html)
It covers all kinds of scenarios. I can keep the SSH agent part, that's the same in any *nix environment.

**Comment by Greg-Szabo** [comment]
1. Setup only tendermint:
Good point. First I wanted to get a working tendermint+basecoin going, so it's still mixed in the config part. I'll work on that next. (I wanted to share what I did so far, hence the pull request. We don't need to merge it until you feel confident.)
Also, I'd like to create packages instead of Ansible playbook for installing tendermint. It should be a one-step process just like installing ansible or any other package on linux.

2. Log files should be managed separately with the Linux OS built-in methods. (logrotate) Also, I've started looking at hosted Elastic solutions to visualize logs for our environments. The playbook for that is being created too.

Currently only Ubuntu 14 was tested as the nodes' operating system and CentOS as Ansible's operating system. I would like to test other scenarios, like the mentioned OSX for Ansible or CentOS for nodes.

File: ansible/README.md (hunk 2 above)
Comment (adrianbrink) [inline ansible/README.md]: I think it would be cool to give a brief explanation for both platforms and then put the link underneath

Comment (Greg-Szabo) [inline ansible/README.md]: Ok, Ansible installation document clarified. I'm still working on separating tendermint and basecoin (and adding ethermint), it should be done sometime early next week.

**Comment by ebuchman** [comment]
This is wonderful. Merging into new branch so we can have in the repo and keep working on it there. Should be ready for master soon :)

**Pull request merged**